packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -5774,6 +5947,7 @@ describeWithDOM('mount', () => {
     in div (created by ErrorBoundary)
     in ErrorBoundary (created by ErrorSFC)
     in ErrorSFC (created by WrapperComponent)
+    in RootFinder (created by WrapperComponent)
```

Comment:
also this one.

Reply:
I'm gonna push back here. This isn't `.debug()` output, it's the stack react 16 prints when an error is thrown. `<RootFinder />` is an enzyme "internal" component, but `<WrapperComponent />`, which is also an internal component _does_ appear in this output. And I think that means `<RootFinder />` technically belongs here too.

Reply:
I understand why it appears there; but *users* should not see the implementation details of enzyme. That there's one wrapper component is unavoidable (or else I'd happily avoid it), but there shouldn't be two.